pages/index.tsx

```diff
@@ -61,92 +61,71 @@ const Home: NextPage = () => {
   if (isLoading) {
     return (
       <div>
-        {isLoading && (
-          <>
-            <Loading loading={true} />
-            <Snowfall color={theme.colors.primary} />
-          </>
-        )}
+        <Loading loading={true} />
+        <Snowfall color={theme.colors.primary} />
       </div>
     );
   } else {
     return (
       <div>
-      {isLoading && (
-        <>
-          <Loading loading={true} />
-          <Snowfall color={theme.colors.primary} />
-        </>
-      )}
-      {!isLoading && (
-        <>
-          <Parallax
-            blur={PARALLAXCONF.blur}
-            bgImage={PARALLAXCONF.bgImage}
-            strength={PARALLAXCONF.strength}
-          >
-            <Section>
-              <Container>
-                <StyledHeading
-                  type="h1"
-                  size="3rem"
-                  bold={true}
-                >
-                  Welcome to the CheaprEats
-                  <br />
-                  Voice Ordering System
-                </StyledHeading>
-                <Button
-                  onClick={redirectToOrder}
-                  icon={BagFill}
-                  primary={true}
-                  iconSize="20px"
-                  margin="5px"
-                >
-                  Start Ordering
-                </Button>
-                <Button
-                  onClick={scrollOnClick}
-                  icon={InfoCircleFill}
-                  iconSize="20px"
-                  margin="5px"
-                >
-                  Learn More
-                </Button>
-              </Container>
-            </Section>
-            <Snowfall color={theme.colors.primary} />
-          </Parallax>
+        <Parallax
+          {...PARALLAXCONF}
+        >
           <Section>
-            <Subsection bgCol={theme.colors.primary}>
-              <Img src={LOGO} />
-            </Subsection>
-            <Subsection>
-              <Heading type="h1" color="black" bold={true}>
-                How it works
-              </Heading>
-              <h1 ref={scrollRef}></h1>
-              <CarouselTestimonial
-                carouselTitle={CAROUSELCONF.title}
-                carouselImage={CAROUSELCONF.image}
-                reviews={CAROUSELCONF.reviews}
-                isAutoplaying={CAROUSELCONF.autoPlay}
-                carouselInterval={CAROUSELCONF.interval}
-                isLooping={CAROUSELCONF.loop}
-              />
+            <Container>
+              <StyledHeading
+                type="h1"
+                size="3rem"
+                bold={true}
```

Comment:
If bold is always true why just create a styled component with the CSS for this always